region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -812,6 +854,71 @@
       "reusableRoomwideNotable": "Pink Hoppers Shinespark into the Wall to Ride the Elevator",
       "note": "Shinesparking horizontally into the wall will trigger the elevator as Samus's echos hit it."
     },
+    {
+      "link": [2, 3],
+      "name": "Pink Hoppers X-Ray Climb Spring Ball Jump",
+      "entranceCondition": {
+        "comeInWithDoorStuckSetup": {}
+      },
+      "requires": [
+        "canXRayClimb",
+        "canTrickyJump",
+        "canLateralMidAirMorph",
+        "canTrickySpringBallJump",
+        "canSpringFling"
+      ],
+      "note": [
+        "X-Ray climb until Samus is a little over halfway off screen; the position is not precise.",
+        "Face right, then do a turnaround buffer spin jump into a Spring Ball jump to reach the ledge.",
+        "The pause input can be pressed at the same time as the turnaround.",
+        "It helps to do an airball to gain horizontal speed.",
+        "The jump input after pausing should not be buffered, in order to allow a little time for the spring fling horizontal speed to take effect."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Pink Hoppers X-Ray Climb HiJump Insane Jump",
+      "entranceCondition": {
+        "comeInWithDoorStuckSetup": {}
+      },
+      "requires": [
+        "HiJump",
+        "canXRayClimb",
+        "canInsaneJump"
+      ],
+      "note": [
+        "Get stuck 3 pixels inside the door, and X-ray climb to a precise height:",
+        "after just one pixel of Samus' feet are visible while crouching facing left, perform exactly 7 more X-Ray stand-ups.",
+        "Use X-ray to face left, then run and jump to barely make it onto the ledge."
+      ],
+      "devNote": [
+        "3 pixels stuck in the door is X position 478 (0x1DE)."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "Pink Hoppers G-Mode Deep Stuck X-Ray Climb Insane Jump",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "canXRayClimb",
+        "canInsaneJump"
+      ],
+      "note": [
+        "Enter with G-Mode and use X-Ray to get stuck 9 pixels inside the door.",
+        "X-Ray climb to a relatively specific height:",
+        "after just one pixel of Samus' feet are visible while crouching facing left, perform between 20 and 22 more X-Ray stand-ups (any of these work).",
+        "Use X-ray to face left, then run and jump to barely make it onto the ledge."
+      ],
+      "devNote": [
+        "9 pixels stuck in the door is X position 484 (0x1E4).",
```

Comment:
482 and 483 looked to work.  485 lets you run out so should work too.

Harder than the HiJump strat because the jump timing is really weird.  More blind standups could be a problem but weren't in my test run.

Reply:
I tried 485 and couldn't get it to work: it did let you run out but it would always bonk when I tried to jump. I saw that some of the positions to the left could also work but I didn't thoroughly test to see if they worked at all the same vertical positions or not.